app/routes/pages/components/Sidebar.css

```diff
@@ -24,9 +23,13 @@
 }
 
 .sidebarHeader {
-  font-size: 2em;
-  margin-bottom: -10px;
+  font-size: 1.8em;
+  margin: 20px 0 0px 0;
```

Comment:
`margin-left` (or right?, don't remeber, hence why it's a good idea to be explicit) ?

Reply:
I believe this is `margin-top`, so `margin-top: 20px` will do. Remember that `stylelint` will complain when setting `0px` instead of just `0`.